tf/src/tf_echo.cpp

```diff
@@ -91,14 +91,25 @@ int main(int argc, char ** argv)
   }
   ros::Rate rate(rate_hz);
 
+  int precision(3);
```

Comment:
There's no need to initialize this here it's confusing to have it here and 3 lines below.

Reply:
Ok